device/mellanox/x86_64-mlnx_msn3800-r0/Mellanox-SN3800-D100C12S2/port_config.ini

```diff
@@ -0,0 +1,115 @@
+# name       lanes            alias      index    speed
+Ethernet0    0,1              etp1a          1    50000
+Ethernet2    2,3              etp1b          1    50000
+Ethernet4    4,5              etp2a          2    50000
+Ethernet4    6,7              etp2b          2    50000
```

Comment:
Why do we have two Ethernet4 defined here?

Reply:
got it  @dgsudharsan, will fix it.